DESCRIPTION

```diff
@@ -16,6 +16,7 @@ Depends:
     R (>= 4.0.0)
 Imports:
     cli (>= 3.6.2),
+    glue,
```

Comment:
I feel like glue is useful when preparing the little code snippets, we usually regard it as "free", and it's already a hard indirect dependency.

``` r
pak::pkg_deps_explain("DavisVaughan/r-tree-sitter", "glue")
#> ℹ Loading metadata database
#> ✔ Loading metadata database ... done
#> 
#> treesitter -> vctrs -> glue
#> treesitter -> vctrs -> lifecycle -> glue
```